src/sagemaker/pytorch/processing.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2019-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""This module contains code related to PyTorch Processors which are used for Processing jobs.
+
+These jobs let customers perform data pre-processing, post-processing, feature engineering,
+data validation, and model evaluation and interpretation on SageMaker.
+"""
+from __future__ import absolute_import
+
+from sagemaker.processing import FrameworkProcessor
+from sagemaker.pytorch.estimator import PyTorch
+
+
+class PyTorchProcessor(FrameworkProcessor):
+    """Handles Amazon SageMaker processing tasks for jobs using PyTorch containers."""
+
+    estimator_cls = PyTorch
+
+    def __init__(
+        self,
+        framework_version,  # New arg
+        role,
+        instance_count,
+        instance_type,
+        py_version="py3",  # New kwarg
+        image_uri=None,
+        command=None,
+        volume_size_in_gb=30,
+        volume_kms_key=None,
+        output_kms_key=None,
+        code_location=None,  # New arg
```

Comment:
I think we can remove these comments  from arguments section

Reply:
comments removed.